README.md

```diff
@@ -440,151 +517,230 @@ Before setting up the following Survey application, dependencies given below sho
 
 2.  **Git Clone Services And Portal Repositories**
 
-   -  **Ubuntu/Linux/MacOS**
+      -  **Ubuntu/Linux/MacOS**
+
+         ```
+         git clone -b main https://github.com/ELEVATE-Project/samiksha-service.git && \
+         git clone -b main https://github.com/ELEVATE-Project/entity-management.git && \
+         git clone -b master https://github.com/ELEVATE-Project/user.git && \
+         git clone -b master https://github.com/ELEVATE-Project/notification.git && \
+         git clone -b main https://github.com/ELEVATE-Project/interface-service.git && \
+         git clone -b master https://github.com/ELEVATE-Project/scheduler.git && \
+         git clone -b main https://github.com/ELEVATE-Project/observation-survey-projects-pwa.git
+         ``` 
+      -  **Windows**
+
+         ```
+         git clone -b main https://github.com/ELEVATE-Project/samiksha-service.git & ^
+         git clone -b main https://github.com/ELEVATE-Project/entity-management.git & ^
+         git clone -b master https://github.com/ELEVATE-Project/user.git & ^
+         git clone -b master https://github.com/ELEVATE-Project/notification.git & ^
+         git clone -b main https://github.com/ELEVATE-Project/interface-service.git & ^
+         git clone -b master https://github.com/ELEVATE-Project/scheduler.git & ^
+         git clone -b main https://github.com/ELEVATE-Project/observation-survey-projects-pwa.git
+         ``` 
+
 
-      ```
-      git clone -b main https://github.com/ELEVATE-Project/samiksha-service.git && \
-      git clone -b main https://github.com/ELEVATE-Project/entity-management.git && \
-      git clone -b master https://github.com/ELEVATE-Project/user.git && \
-      git clone -b master https://github.com/ELEVATE-Project/notification.git && \
-      git clone -b main https://github.com/ELEVATE-Project/interface-service.git && \
-      git clone -b master https://github.com/ELEVATE-Project/scheduler.git && \
-      git clone -b main https://github.com/ELEVATE-Project/observation-survey-projects-pwa.git
-      ``` 
 
 3.  **Install NPM Packages**
 
-   -  **Ubuntu/Linux/MacOS**
+      -  **Ubuntu/Linux/MacOS**
+
+         ```
+         cd samiksha-service && npm install && cd ../ && \
+         cd entity-management/src && npm install && cd ../.. && \
+         cd user/src && npm install && cd ../.. && \
+         cd notification/src && npm install && cd ../.. && \
+         cd interface-service/src && npm install && cd ../.. && \
+         cd scheduler/src && npm install && cd ../.. && \
+         cd observation-survey-projects-pwa && npm install --force && cd ..
+         ```  
+      -  **Windows**
+
+         ```
+         cd samiksha-service & npm install & cd ..\ & ^
+         cd entity-management\src & npm install & cd ..\.. & ^
```

Comment:
@MallanagoudaB is entity-management service working for any node version OR only for a specific version. If it works only for a specific node version. Please mention it separately